irods/test/harness/tests/repo

```diff
@@ -0,0 +1 @@
+../../../..
```

Comment:
To figure out relative paths when generating the execute line for the docker container

Reply:
This no longer exists. The purpose is now fulfilled by  [print_repo_root_location](https://github.com/d-w-moore/python-irodsclient/blob/502.m/irods/test/harness/print_repo_root_location), a script which prints the absolute path of the repo regardless the current working directory when it's launched.  That absolute result is useful within [docker_container_driver.sh](https://github.com/d-w-moore/python-irodsclient/blob/502.m/irods/test/harness/docker_container_driver.sh) especially when calculating the paths of things relative to that root.

Reply:
Let me know if this seems an adequate explanation, and I'll resolve.  But, of course, I will understand if the truth of the matter is otherwise.  It was confusing to write, probably more confusing to read. The end purpose is calculating _container-internal_ paths using those _container-external_ paths as input, which was less trivial than I'd first imagined.